pkg/background/update_request_controller.go

```diff
@@ -370,6 +370,50 @@ func (c *controller) deletePolicy(obj interface{}) {
 	}
 }
 
+func (c *controller) deleteNSPolicy(obj interface{}) {
+	p, ok := kubeutils.GetObjectWithTombstone(obj).(*kyvernov1.Policy)
+	if !ok {
+		logger.Info("Failed to get deleted object", "obj", obj)
+		return
+	}
+
+	logger.V(4).Info("deleting policy", "name", p.Name)
+	key, err := cache.MetaNamespaceKeyFunc(kubeutils.GetObjectWithTombstone(obj))
+	if err != nil {
+		logger.Error(err, "failed to compute policy key")
```

Comment:
error message can be changed here. instead of `compute` you can say `load` or `find`

Reply:
fixed